src/Team-Services-Bot.AcceptanceTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Team-Services-Bot.AcceptanceTests")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Team-Services-Bot.AcceptanceTests")]
+[assembly: AssemblyCopyright("Copyright ©  2017")]
+[assembly: AssemblyTrademark("")]
```

Comment:
I think we should link to a CommonAssemblyInfo.cs file rather than duplicate common attributes